lib/src/driver.dart

```diff
@@ -64,16 +64,23 @@ class Driver {
 
   bool silent = false;
 
+  final String? sdkPath;
+
   /// Handles printing.  Can be overwritten by clients.
   Logger logger = Logger.standard();
 
-  Driver(ArgResults argResults)
-      : options = CommandLineOptions.fromArgs(argResults),
+  Driver(
+    ArgResults argResults, {
+    this.sdkPath,
+  })  : options = CommandLineOptions.fromArgs(argResults),
         sources = argResults.rest
             .map((p) => path.normalize(io.File(p).absolute.path))
             .toList();
 
-  factory Driver.forArgs(List<String> args) {
+  factory Driver.forArgs(
+    List<String> args, {
+    String? sdkPath,
```

Comment:
Should the sdk path be a command-line argument?

Reply:
Good thinking.

/fyi @polina-c